### PR TITLE
Filter using like and CSV working with array/boolean fields

### PR DESCRIPTION
**Pull request by luiztiago**

### CODE REVIEW

**Comment by dcatipon** [comment]
Looks good to me! 👍  :shipit:

File: Controller/AbstractCoreCRUDController.php [hunk 1]
```diff
@@ -232,7 +231,16 @@ public function handleCsv(Request $request, array $criteria = [])
                 foreach($fields as $field) {
                     $getField = 'get' . ucfirst($field);
                     if (method_exists($entity, $getField)) {
-                        $row[] = $entity->$getField();
+                        $value = $entity->$getField();
+                        if (is_array($value)) {
+                            $value = implode(',', $value);
+                        }
+                        $row[] = $value;
+                        continue;
```
Comment (johnmadrak) [inline Controller/AbstractCoreCRUDController.php]: @luiztiago - Can you add a blank line before continue?

File: Controller/AbstractCoreCRUDController.php [hunk 2]
```diff
@@ -232,7 +231,16 @@ public function handleCsv(Request $request, array $criteria = [])
...
+                    }
+                    $isField = 'is' . ucfirst($field);
```
Comment (johnmadrak) [inline Controller/AbstractCoreCRUDController.php]: @luiztiago - Can you add a blank line before $isField?

**Comment by luiztiago** [comment]
@johnmadrak done :)

**Comment by johnmadrak** [comment]
:+1: :ship:

**Pull request merged**